### ISSUE TITLE
Bug: "Copy Invoice" uncaught exception in "Buy an Instant Channel"

### ISSUE THREAD

**Issue by MegalithicBTC** [getAlby/nostr-wallet-connect-next#456]
Please see screenshot below.

I would like to try to fix this issue, however I could use a little bit of direction here. 

Error is happening here:

https://github.com/getAlby/bitcoin-connect/blob/b1306775814135ff7fe14e0e4758d60286321cf1/src/components/pages/bc-send-payment.ts#L213

Would you suggest that I dig into the `bitcoin-connect` package to try to debug this issue? Or are we using `bitcoin-connect` incorrectly?

![copy-invoice-failure](https://github.com/getAlby/nostr-wallet-connect-next/assets/162853102/60d0919b-fb3a-4bf5-b0ad-25510df46a9d)

**Comment by bumi** [comment]
In what setup are you running the app? it's likely on a non https setup?

@im-adithya you had a workaround for this case, didn't you? can you add that to bitcoin-connect, too?

**Comment by MegalithicBTC** [comment]
Yes, was a non-http setup. Closing this as it might be not be worth fixing this edge case.

**Comment by bumi** [comment]
actually it would be good to solve. if you run it at home or just on a private IP then this should also work.

**Comment by rolznz** [comment]
@im-adithya should have fixed this about a week ago in https://github.com/getAlby/nostr-wallet-connect-next/pull/440

@MegalithicBTC did you have the latest changes?

**Comment by im-adithya** [comment]
Yeah, please let us know if it breaks even after the update!
